README.md

```diff
@@ -90,19 +90,118 @@ The load operation may take up to a few of seconds. Load Ver-ID using the `load`
     }
 ~~~
 
+## Register and Authenticate User From Flutter
+The Ver-ID Person plugin module will be available in your script from the import class VerID.  The rest of the support classes can be imported from the main package via the route package:veridflutterplugin/src/ :
 
+~~~dart
+  //method for user registration
+  Future<SessionResult> registerUser(String userId) async {
+    if (verID != null) {
+      RegistrationSessionSettings settings =
+          new RegistrationSessionSettings(userId: userId);
+      settings.showResult = true;
+      return VerID.register(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+  
+  Future<SessionResult> registerUser(String userId) async {
+    if (verID != null) {
+      RegistrationSessionSettings settings =
+          new RegistrationSessionSettings(userId: userId);
+      settings.showResult = true;
+      return VerID.register(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
+
+### Liveness Detection
+
+In a liveness detection session the user is asked to assume a series of random poses in front of the camera.
 
-### Face detection session without asking for poses
+Liveness detection sessions follow the same format as registration and authentication.
 
-### Liveness detection session defining the bearings (poses) the user may be asked to assume
+## Extracting faces for face comparison
+
+~~~dart
+Future<SessionResult> captureLiveFace() async {
+    if (verID != null) {
+      LivenessDetectionSessionSettings settings =
+          new LivenessDetectionSessionSettings();
+      return VerID.captureLiveFace(settings: settings).then((value) {
+        if (value == null) {
+          throw 'Session canceled';
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
+## Face detection session without asking for poses
```

Comment:
The sample code in this section is identical to the code in the previous section.